lib/presentation/onboarding/onboard.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:collaction_app/presentation/core/collaction_icons.dart';
+import 'package:flutter/material.dart';
+
+import '../themes/constants.dart';
+
+/// Single onboarding slate
+class OnBoard extends StatelessWidget {
+  /// Icon to display in board
+  final IconData icon;
+
+  /// Onboard title
+  final String title;
+
+  /// Onboard message
+  final String message;
+
+  /// Screen scale factor
+  final double scaleFactor;
+
+  const OnBoard({
```

Comment:
This class has a way too ambiguous name. You can name it `OnboardingStep` (Onboard is one word)